clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/loadbalancer/view/AzureLoadBalancerProvider.groovy

```diff
@@ -76,7 +77,8 @@ class AzureLoadBalancerProvider implements LoadBalancerProvider<AzureLoadBalance
       account: parts.account?: "none",
       name: loadBalancerDescription.loadBalancerName,
       region: loadBalancerDescription.region,
-      vnet: loadBalancerDescription.vnet?: "none"
+      vnet: loadBalancerDescription.vnet?: "vnet-unassigned",
+      serverGroups: [new LoadBalancerServerGroup(name: loadBalancerDescription.serverGroup, isDisabled: false, detachedInstances: [], instances: [])]
```

Comment:
In the Deck UI we get the relationship between load balancer and server groups that are attached to by just creating and populating the LoadBalancerServerGroup.